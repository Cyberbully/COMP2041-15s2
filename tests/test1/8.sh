#!/bin/sh

# Enter pipeline/shell script below.
