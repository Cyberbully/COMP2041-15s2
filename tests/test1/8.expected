232 q8/ hKnuk25i3oSbeUiO YolKTxu3EWXBf51g23M1G5eREmU5
173 q8/2OXZjl3sDJH s2roTuQUAEWdGC23hgOX
46 q8/JW C5agUOCaTU
197 q8/KG w5kMQaOmhF4q5 nf bgdQyqLugbM33rDS3P5ldskdst5ax2K 4wz6Qs2l3S3 v55zWavatrIU
138 q8/QUvla6Y63GXLVtoih5PfL3ecnUGMLHD3 o26Q31kHyC25Ev 54CMER OT T5kRS 6ASj2mBdDnOy
280 q8/TOFYzEEeAURk5 3W QM33NGNIsLBJxOTWQPjVKe QaVzItkoagUh1I fwcVokBgmGs2Amvq6MTLnh PSbkYCxO2 OaICiPX
17 q8/boohoo.py
87 q8/file has space in name.2041
213 q8/j 2JShyPE3 UfAXJtGCd
25 q8/testfile2_1
72 q8/testfile2_2
230 q8/testfile2_3
178 q8/testfile2_4
178 q8/testfile2_5
186 q8/testfile2_6
115 q8/testfile2_9
230 q8/testfile2_file has space in name.2041
